### PR TITLE
Added Set Image Pull Policy To Always query #553

### LINKED ISSUE #553: Add Set Image Pull Policy Of The Container To Always query for Kubernetes
### Platform
Kubernetes

### Description
Image Pull Policy of the container must be defined and set to Always

### PR DESCRIPTION
**Pull request by joelcarvalhocheckmarx**
Image Pull Policy of the container must be defined and set to Always

Closes #553

### CODE REVIEW

File: assets/queries/k8s/resource_with_allow_privilege_escalation/image_pull_policy_of_container_to_always/query.rego [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+CxPolicy [ result ] {
+   spec := input.document[i].spec 
+   containers := spec.containers
+   containers[c].imagePullPolicy != "Always"
+   
+   not contains(containers[c].image, ":latest")
+   
+   result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("spec.containers[%d].imagePullPolicy", [c]),
```
Comment (joaomartinscx) [inline assets/queries/k8s/resource_with_allow_privilege_escalation/image_pull_policy_of_container_to_always/query.rego:12]: missing metadata tags, please update

File: assets/queries/k8s/resource_with_allow_privilege_escalation/image_pull_policy_of_container_to_always/metadata.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "set_image_pull_policy_of_the_container_to_always",
+  "queryName": "Set Image Pull Policy Of The Container To Always",
```
Comment (joaomartinscx) [inline assets/queries/k8s/resource_with_allow_privilege_escalation/image_pull_policy_of_container_to_always/metadata.json:3]: This is only a comment, this is not required, but I would rephrase the query name to identify the failure, instead of the expected value, and specify the expected value in keyExpectedValue

**Review by joaomartinscx — CHANGES_REQUESTED** [review]

**Review by joaomartinscx — COMMENTED** [review]
Everything looks good to me, I recommend the approval and merge of this PR

**Review by ruigomescx — APPROVED** [review]
